.gitignore

```diff
@@ -0,0 +1,36 @@
+.DS_Store
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**/build/
+!**/src/test/**/build/
+gradlew
+gradlew.bat
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
```

Comment:
What is the difference between a class path and factory path?

Reply:
The classpath is used at runtime to point to the compiled classes. I'm not sure what the factory path is - this .gitignore was one of the default ones you can select when working on a java project. Many of the files referenced in this gitignore won't even be relevant to our project but it's okay because those references don't hurt us in any way so we can keep them.